day_7/checker_board.rb

```diff
@@ -0,0 +1,18 @@
+board = Array.new(6) {Array.new(6)}
+row = 0
+board.each do |i|
+  col = 0
+
+  i.each do |j|
+    if col % 2 == 0 && row % 2 == 0
+      print "X"
+    elsif col % 2 == 1 && row % 2 == 1
+      print "X"
+    else
+      print " "
+    end
+    col += 1
+  end
+  row += 1
+  print "\n"
+end
```

Comment:
Excellent results here! Again, I think you can explore getting some user input. I'm also wondering whether you need to create an array to iterate over every element of that array, or if you could just do something a certain amount of times... I think you could avoid the array creation, but perhaps there is something I'm missing.